### PR TITLE
Finish supplier register [TAS-84]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

**Comment by femacedogomes** [comment]
still with console.log on handlesubmit because we dont have endpoint for that

File: src/components/atoms/CompleteSupplierRegister/index.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      .required('Ensira uma logo para seu estabelecimento')
+      .test("FILE_SIZE", "Arquivo muito grande! Selecione um de menor tramanho", (value: any) => !value || (value && value.size <= 1024 * 1024))
+      .test("FILE_FORMAT", "Arquivo com formato não suportado", (value: any) => !value || (value && ['image/png', 'image/jpeg'].includes(value.type))),
+    description: Yup.string().required("escolha uma descrição para seu estabelecimento")
+  });
+
+  const handleFormSubmit = async (values: ISupplierCompleteRegister) => {
+    setloading(true)
+    //criar endpoint para atualizar cadastro supplier
+    console.log(values)
```
Comment (renanpontez) [inline src/components/atoms/CompleteSupplierRegister/index.tsx:35]: +1 protein bar

File: src/components/atoms/CompleteSupplierRegister/index.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  const [loading, setloading] = useState(false);
+  const [logoPlaceHolder, setLogoPlaceHolder] = useState('...carregar')
+  const [ilustrationImagePlaceHolder, SetIlustrationImagePlaceHolder] = useState('...carregar')
+
+
+  const CompleteSupplierRegisterSchema = Yup.object().shape({
+
+    logo: Yup.mixed()
+      .nullable()
+      .required('Ensira uma logo para seu estabelecimento')
```
Comment (pedrohbl03) [inline src/components/atoms/CompleteSupplierRegister/index.tsx:21]:
```suggestion
      .required('Insira uma logo para seu estabelecimento')
```

File: src/components/atoms/Textarea/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { TextareaHTMLAttributes } from "react";
+
+export interface TextareaProps
+  extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+    field?: any
+  }
+
+// eslint-disable-next-line react/display-name
```
Comment (pedrohbl03) [inline src/components/atoms/Textarea/index.tsx:8]: Remove this and add before `export default Textarea;` -> Textarea.DisplayName = 'Textarea'

**Review by pedrohbl03 — APPROVED** [review]
That's nice. But, solve the minimal problems pointed out in the comments and Resolve merge conflicts before merge.

**Pull request merged**